subsys/net/lib/mqtt_helper/mqtt_helper.c

```diff
@@ -14,33 +14,36 @@
 #else
 #include <zephyr/net/socket.h>
 #endif /* CONFIG_POSIX_API */
-#include <net/azure_iot_hub.h>
-#include <net/azure_iot_hub_dps.h>
 
-#include "azure_iot_hub_mqtt.h"
+#include <mqtt_helper.h>
+#include <zephyr/net/mqtt.h>
 
-#if defined(CONFIG_AZURE_IOT_HUB_PROVISION_CERTIFICATES)
-#include CONFIG_AZURE_IOT_HUB_CERTIFICATES_FILE
-#endif /* CONFIG_AZURE_IOT_HUB_PROVISION_CERTIFICATES */
+#if defined(CONFIG_MQTT_HELPER_PROVISION_CERTIFICATES)
+#include CONFIG_MQTT_HELPER_CERTIFICATES_FILE
```

Comment:
I think we should actually have `#warning ` message here if private key is defined

Reply:
Good idea (y)

Reply:
Something like this?

```
#warning "Run-time provisioning of credentials is enabled. This option is only intended for testing"
	 " purposes and should be avoided in a production scenario. This is due to the credentials"
	 " being exposed in flash memory."
```

Reply:
Please see the current suggestion for such a comment.

Reply:
Yeap, that looks good